es-bs-base/scss/variables/_grays.scss

```diff
@@ -0,0 +1,18 @@
+/* stylelint-disable selector-pseudo-class-no-unknown, property-no-unknown */
+
+@import "../bootstrap";
+@import "../variables";
+
+:export {
+  white: #fff;
+  gray-100: #fafafa;
+  gray-200: #ececec;
+  gray-300: #dddddd;
+  gray-400: #bababa;
+  gray-500: #999999;
+  gray-600: #7e7e7e;
+  gray-700: #565656;
+  gray-800: #404040;
+  gray-900: #292929;
+  black: #000;
```

Comment:
Shouldn't these reference the variable names and not the hex values?

Reply:
This is for building out the gray color pallet, there was an issue with importing variables. I'm just moving things from extends, but should probably revisit.